cmd/ci-operator-prowgen/main.go

```diff
@@ -102,13 +103,29 @@ func generatePodSpec(info *config.Info, target string, additionalArgs ...string)
 				Image:           "ci-operator:latest",
 				ImagePullPolicy: kubeapi.PullAlways,
 				Command:         []string{"ci-operator"},
-				Args:            append([]string{"--give-pr-author-access-to-namespace=true", "--artifact-dir=$(ARTIFACTS)", fmt.Sprintf("--target=%s", target)}, additionalArgs...),
-				Env:             []kubeapi.EnvVar{{Name: "CONFIG_SPEC", ValueFrom: &configMapKeyRef}},
+				Args: append([]string{
+					"--give-pr-author-access-to-namespace=true",
+					"--artifact-dir=$(ARTIFACTS)",
+					fmt.Sprintf("--target=%s", target),
+					"--sentry-dsn-path=/etc/sentry-dsn/ci-operator",
+				}, additionalArgs...),
+				Env: []kubeapi.EnvVar{{Name: "CONFIG_SPEC", ValueFrom: &configMapKeyRef}},
 				Resources: kubeapi.ResourceRequirements{
 					Requests: kubeapi.ResourceList{"cpu": *resource.NewMilliQuantity(10, resource.DecimalSI)},
 				},
+				VolumeMounts: []kubeapi.VolumeMount{{
+					Name:      sentryDsnName,
+					MountPath: "/etc/sentry-dsn",
```

Comment:
Why not share this with the flag string?

Reply:
Yeah, I extracted everything to constants.